### DISCUSSION TITLE
error on start couchdb

### DISCUSSION THREAD

**Discussion by diegonella** [apache/couchdb#3107] [Q&A]
Hi, 
I have, ubuntu 14 and couchdb 1.6,
the problem is that I don't know where to start checking.
I have no configuration changes, or anything similar.
The problem started after the server ran out of space

When try start service, return
[Thu, 27 Aug 2020 23:50:44 GMT] [error] [<0.128.0>] {error_report,<0.30.0>,
                     {<0.128.0>,crash_report,
                      [[{initial_call,{couch_db_updater,init,['Argument__1']}},
                        {pid,<0.128.0>},
                        {registered_name,[]},
                        {error_info,
                         {exit,
                          {{badrecord,db_header},
                           [{couch_db_updater,init_db,6,
                             [{file,"couch_db_updater.erl"},{line,441}]},
                            {couch_db_updater,init,1,
                             [{file,"couch_db_updater.erl"},{line,46}]},
                            {gen_server,init_it,6,
                             [{file,"gen_server.erl"},{line,304}]},
                            {proc_lib,init_p_do_apply,3,
                             [{file,"proc_lib.erl"},{line,239}]}]},
                          [{gen_server,init_it,6,
                            [{file,"gen_server.erl"},{line,328}]},
                           {proc_lib,init_p_do_apply,3,
                            [{file,"proc_lib.erl"},{line,239}]}]}},
                        {ancestors,[<0.127.0>,<0.125.0>]},
                        {messages,[]},
                        {links,[<0.127.0>]},
                        {dictionary,[]},
                        {trap_exit,true},
                        {status,running},
                        {heap_size,610},
                        {stack_size,27},
                        {reductions,513}],
                       [{neighbour,
                         [{pid,<0.125.0>},
                          {registered_name,[]},
                          {initial_call,{erlang,apply,2}},
                          {current_function,{proc_lib,sync_wait,2}},
                          {ancestors,[]},
                          {messages,[]},
                          {links,[<0.126.0>,<0.127.0>,<0.87.0>]},
                          {dictionary,[]},
                          {trap_exit,false},
                          {status,waiting},
                          {heap_size,233},
                          {stack_size,9},
                          {reductions,50}]},
                        {neighbour,
                         [{pid,<0.127.0>},
                          {registered_name,[]},
                          {initial_call,{couch_db,init,['Argument__1']}},
                          {current_function,{proc_lib,sync_wait,2}},
                          {ancestors,[<0.125.0>]},
                          {messages,
                           [{ack,<0.128.0>,
                             {error,
                              {{badrecord,db_header},
                               [{couch_db_updater,init_db,6,
                                 [{file,"couch_db_updater.erl"},{line,441}]},
                                {couch_db_updater,init,1,
                                 [{file,"couch_db_updater.erl"},{line,46}]},
                                {gen_server,init_it,6,
                                 [{file,"gen_server.erl"},{line,304}]},
                                {proc_lib,init_p_do_apply,3,
                                 [{file,"proc_lib.erl"},{line,239}]}]}}}]},
                          {links,[<0.125.0>,<0.128.0>]},
                          {dictionary,[]},
                          {trap_exit,false},
                          {status,runnable},
                          {heap_size,376},
                          {stack_size,16},
                          {reductions,43}]}]]}}

**Comment by janl** [comment]
> {badrecord,db_header}

sounds like one of your databases got corrupted because of the no more space issue.

Note that we don’t support CouchDB 1.x anymore. You must upgrade.
